src/app/Caches/Create/BasicCacheConfigConfigurator.tsx

```diff
@@ -215,8 +215,8 @@ const BasicCacheConfigConfigurator = () => {
             id="statistics"
             isChecked={isStatistics}
             onChange={() => setIsStatistics(!isStatistics)}
-            label={t('caches.create.configurations.basic.statistics-disable')}
-            labelOff={t('caches.create.configurations.basic.statistics-enable')}
+            label={t('caches.create.configurations.basic.statistics-enable')}
+            labelOff={t('caches.create.configurations.basic.statistics-disable')}
             hasCheckIcon
```

Comment:
If you're using a switch with a label (that's the default if there is enough space for it), it's not necessary to include the check icon